Automattic/jetpack
# Payments Block: Remove notice from canvas

Remove the notice in the canvas that often appears unreadable that links to fee structure for payments. Instead add this to the sidebar for the block and make it persistent. This also helps better support block patterns that will use this block.

**Before**

<img width="726" alt="Screen Shot 2020-11-02 at 12 52 36 PM" src="https://user-images.githubusercontent.com/1464705/97918141-7a14e400-1d0a-11eb-9e55-d45633734991.png">

**After**

<img width="1146" alt="Screen Shot 2020-11-02 at 12 51 09 PM" src="https://user-images.githubusercontent.com/1464705/97918155-813bf200-1d0a-11eb-96b4-0f7af11938e0.png">

Partially fixes https://github.com/Automattic/wp-calypso/issues/46884

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:






* Check out this branch locally.
* Sync this branch to your WordPress.com sandbox.
* Sandbox a WordPress.com site
* On this site insert a payments button
* Confirm that you do not see the message in the canvas as appears in the before picture above.
* Confirm that you see the message in the sidebar when the payments block is selected.

#### Proposed changelog entry for your changes:

* Fixed an issue with the Payments block and an unreadable notice.
